packages/serverless/src/backend_connector.js

```diff
@@ -117,6 +117,10 @@ exports.setLogger = function setLogger(_logger) {
  * the variable is always true.
  */
 exports.sendBundle = function sendBundle(bundle, finalLambdaRequest, callback) {
+  logger.debug(
+    `Sending bundle to Instana (no. of spans: ${(bundle && bundle.spans && bundle.spans.length) || 'unknown'})`
```

Comment:
s: can we use optional chaining here? IMO that might be an easy syntax
`Sending bundle to Instana (no. of spans: ${bundle?.spans?.length ?? 'unknown'})`

Reply:
Yes thank you. Will change!